### DISCUSSION TITLE
Include license disclaimer on Terms and Conditions page

### DISCUSSION THREAD

**Discussion by Traveler3568** [communitypoweredbydlot/community-website#46] [Ideas]
The disclaimer regarding the Creative Commons licence might be overwhelming for users, as many are not interested in copying/sharing the content or do not understand the terms of copyright, license, etc

As a common practice, information regarding the owner of a website, as well as license of the content should be included in Terms and conditions. 

This will help us keep a clean visual of the website by featuring only important information on the main sections and maintain a good amount of white space.   

**RO**
Website-ul  https://www.asociatiacommunity.ro/ este proprietatea Asociației Community powered by DLOT, o asociaţie cu personalitate juridică, neguvernamentală, apolitică şi non-profit, care se supune legislaţiei române, aplicabilă acestui gen de persoană juridică.
Cu excepția locurilor în care se menționează altfel, conținutul acestui site este licențiat sub o licență Creative Commons Attribution Share Alike 4.0 International.

**EN**
The website https://www.asociatiacommunity.ro/ is proprety of Community powered by DLOT, a non-governmental, apolitical and non-profit Organization which is subject to Romanian legislation, applicable to this type of legal persona. 
Except where otherwise noted, content on this site is licensed under a Creative Commons Attribution Share Alike 4.0 International

**Comment by archblob** [comment]
I'd say let's sit on this and see if it ever becomes an issue.

I see no evidence _at the moment_ that this is or could be overwhelming for someone.
Regarding the clean visual style I again see no reason to introduce a change _at the moment_ as it's not evident to me that it's an improvement.Personally I consider licensing to be a pretty important piece of information. Examples of how just removing this improves the website are welcome.

I think we should consider other options first, like centering it and moving it at the bottom like we can see on mobile or any other ideas you might have.
